lib/logstash/codecs/json_lines.rb

```diff
@@ -53,7 +53,8 @@ def encode(event)
   # from_json_parse uses the Event#from_json method to deserialize and directly produce events
   def from_json_parse(json, &block)
     LogStash::Event.from_json(json).each { |event| yield event }
-  rescue LogStash::Json::ParserError
+  rescue LogStash::Json::ParserError => e
+    @logger.error("JSON parse error, original data now in message field", :error => e, :data => json)
```

Comment:
Fair point. I'll stick to a warning since that matches the default log level. Hiding what should be obvious signs of trouble behind a `--verbose` invocation seems ill-advised.